smartmin/views.py

```diff
@@ -1388,16 +1390,18 @@ def view_for_action(self, action):
                     view.link_url = 'id@%s' % self.url_name_for_action('read')
                 elif 'update' in self.actions:
                     view.link_url = 'id@%s' % self.url_name_for_action('update')
-                else:
-                    view.link_fields = ()
```

Comment:
Shouldn't clear this just because link_url is undefined - dev may provide their own link URLs by overriding `lookup_field_link `